test/Renci.SshNet.Tests/Classes/Connection/DirectConnectorTest_Connect_TimeoutConnectingToServer.cs

```diff
@@ -60,21 +62,34 @@ protected override void Act()
             {
                 _actualException = ex;
             }
+            catch (SocketException ex)
+            {
+                _actualException = ex;
+            }
             finally
             {
                 _stopWatch.Stop();
             }
         }
 
-        [TestMethod]
-        public void ConnectShouldHaveThrownSshOperationTimeoutException()
+        [TestMethodForPlatform(nameof(OSPlatform.Windows))]
+        public void ConnectShouldHaveThrownSshOperationTimeoutExceptionOnWindows()
         {
             Assert.IsNull(_actualException.InnerException);
+            Assert.IsInstanceOfType<SshOperationTimeoutException>(_actualException);
             Assert.AreEqual(string.Format(CultureInfo.InvariantCulture, "Connection failed to establish within {0} milliseconds.", _connectionInfo.Timeout.TotalMilliseconds), _actualException.Message);
         }
 
-        [TestMethod]
-        public void ConnectShouldHaveRespectedTimeout()
+        [TestMethodForPlatform(nameof(OSPlatform.Linux))]
+        public void ConnectShouldHaveThrownSshOperationTimeoutExceptionOnLinux()
```

Comment:
Is the name intended or is it meant to be:

```suggestion
        public void ConnectShouldHaveThrownSocketExceptionOnLinux()
```

? Same for some other files

Reply:
I will revert this.